test/parallel/test-eslint-lowercase-name-for-primitive.js

```diff
@@ -23,19 +23,23 @@ new RuleTester().run('lowercase-name-for-primitive', rule, {
   invalid: [
     {
       code: 'new errors.TypeError("ERR_INVALID_ARG_TYPE", "a", "Number")',
-      errors: [{ message: 'primitive should use lowercase: Number' }]
+      errors: [{ message: 'primitive should use lowercase: Number' }],
+      output: 'new errors.TypeError("ERR_INVALID_ARG_TYPE", "a", "number")'
```

Comment:
CI failed due to a small problem:
The input code and expected output are using double quotes `"`, but the autofixer will change it to single quotes, which makes the actual output cannot match the expected output.

Reply:
@shobhitchittora Could you change these double quotes in `code` and `output` to using single quotes?

Reply:
@starkwang pushed a fix. How did u check the problem in Jenkins that the output is not correct?

Reply:
I modified some codes in eslint (eslint/lib/testers/rule-tester.js) to expose the error message.  
You can see https://github.com/eslint/eslint/pull/9769 for more detail. 😉